Demo/src/main/java/com/braintreepayments/demo/PayPalFragment.java

```diff
@@ -128,6 +129,13 @@ private void launchPayPal(
                     } else if (paymentAuthRequest instanceof PayPalPaymentAuthRequest.ReadyToLaunch){
                         PayPalPendingRequest request = payPalLauncher.launch(requireActivity(),
                                 ((PayPalPaymentAuthRequest.ReadyToLaunch) paymentAuthRequest));
+
+                        String pairingId = ((PayPalPaymentAuthRequest.ReadyToLaunch) paymentAuthRequest).getRequestParams().getPairingId();
+
+                        if (pairingId != null && !pairingId.isEmpty()) {
+                            Toast.makeText(getActivity(), pairingId, Toast.LENGTH_LONG).show();
```

Comment:
```suggestion
                            Toast.makeText(getActivity(), "Pairing ID: $pairingId", Toast.LENGTH_LONG).show();
```

Adding a label to the value may give more context.